test/gtest/uct/test_md.cc

```diff
@@ -66,6 +66,10 @@ void* test_md::alloc_thread(void *arg)
 ucs_status_t test_md::reg_mem(unsigned flags, void *address, size_t length,
                               uct_mem_h *memh_p)
 {
+    /* Register memory respecting MD reg_alignment */
+    size_t reg_align  = ucs_max(md_attr().reg_alignment, 1);
```

Comment:
1. same here, need to fix UCT MD if it returns 0
2. we should make sure the test allocates aligned memory buffer and not just align during registration. Otherwise, we could try to register pages not in the process address space.

Reply:
1. ok
2. hmm, are we absolutely sure about this?
Looking at [production code](https://github.com/openucx/ucx/blob/master/src/ucp/core/ucp_mm.c#L525), it seems we do exactly that, or I misunderstood it?
Finally we just register the whole page, so it must be within the process address space.

Reply:
2. ok